templates/geosafe/analysis/create.html

```diff
@@ -484,6 +484,38 @@ <h4 class="modal-title" id="myModalLabel">{% trans "Upgrade Your Browser" %}</h4
     </div>
 {% endblock %}
 
+    {% if not user.is_authenticated %}
```

Comment:
CMIIW, @boney-bun is this coming from Geonode's base template?
If that's the case, then I think you don't have to add it again because create.html inherits from geonode_base.html (with some overrides for geosafe) and then geonode_base.html overrides base.html.

Just verify it again for me, if you delete this whole block, I think your code should still working, because this block is already there.